### ISSUE TITLE
Groups and allocation key

### ISSUE THREAD

**Issue by Elvis2000NC** [eneiluj/cospend-nc#133]
Is it possible to join members to different "families"/groups? We are two families on vacation together. Once 3 people, once 4 people. All 7 people are shopping. I would now like to combine each of the family members into one family and see their total spending. Then I want to divide the total spending by 7 people and divide it again among the families.
Question 1: Is it possible to combine single members to a family?
Question 2: Where or how can I choose/enter the allocation key?
I thought I had already posted the family group question here, but I can't find the issue anymore. Maybe I didn't submit it. If it is duplicated, sorry.

Translated with www.DeepL.com/Translator (free version)

**Comment by cestmoimeme** [comment]
Hi
If I understand what you want ,I think you can use the "weight" when create a new user.
If in you case you create a family-one user and give it 4 as a weight , and family-two with 4 as weight, you will have 2 users, one paying for 4 , the other paying for 3

I use it sometimes and it works perfectly 
TO show you howto I created a test with "toto" paying for 2 persons ...  My image is in french but I guess you will understand it 
<img width="378" alt="image" src="https://user-images.githubusercontent.com/17205394/164885726-93b690bd-c6e3-4d60-99b1-f2d3a9850ddd.png">

**Comment by Elvis2000NC** [comment]
Yes, that could be a solution. is it correct, that all members together 100% . the weight is in %? or number of persons?

**Comment by Elvis2000NC** [comment]
this is my list:
![grafik](https://user-images.githubusercontent.com/76623282/164889478-fbf56d4e-9df1-432a-9db7-6013143c0a5f.png)

that`s only the parents and now i want to make only two "families", aber i cannot delete persons like maren and katharina, only react? how can i delete personsS?

**Comment by cestmoimeme** [comment]
Well , it's not a % neither a number of person (even if most the time, it's number of person)
By derfault , the weight is 1
Family-one has 4 persons so you type 4 as weight
But ... imagine a family is one adult and one baby , and you do no want the baby to be like an adult : you can type as weight 1,5 for example (1 for the adult, 0.5 for the baby)

You can do as you want (another example, says 2 adult and a teenager who is 75% of an adult ... the weight will be 2.75 )
and so on ...

**Comment by cestmoimeme** [comment]
> this is my list: ![grafik](https://user-images.githubusercontent.com/76623282/164889478-fbf56d4e-9df1-432a-9db7-6013143c0a5f.png)
> 
> that`s only the parents and now i want to make only two "families", aber i cannot delete persons like maren and katharina, only react? how can i delete personsS?

the best to my opinion is to export in a file your data, change name and merge for a family from the parents and  then import it
